pages/common/assets/sass/style.scss

```diff
@@ -77,3 +77,16 @@
     background: $govuk-error-colour;
   }
 }
+
+.panel-list li {
+  font-size: inherit;
+  font-weight: inherit;
+  line-height: inherit;
+}
```

Comment:
is this a thing?

Reply:
yes, the dashboard is rendered as <ul class="panel-list">

Reply:
I'm just wondering why we need the `inherit` properties on the font? surely these are just no-op's?

Reply:
Actually, now after I have removed the extra divs and some styles, it turns out that we don't. Removed that as well.